README.md

```diff
@@ -138,22 +215,67 @@ gaiacli account <destination_address>
 gaiacli account <your_address>
 ```
 
+You can also check your balance at a given block by using the `--block` flag:
+
+```
+gaiacli account <your_address> --block=<block_height>
+```
+
+## Run a Validator Node
 
-## Staking: Becoming a Validator
+[Validators](https://cosmos.network/validators) are actors from the network that are responsible from commiting new blocks to the blockchain by submitting their votes. This section covers the instructions necessary to stake tokens to become a testnet validator candidate.
```

Comment:
Missing little reminder for people that intend to run validators in the future mainnet that sentry node architecture is strongly recommended